### PR TITLE
Pluralize fields resource name

### PR DESCRIPTION
**Pull request by kureuil**
According to the spec (http://jsonapi.org/format/#fetching-sparse-fieldsets), the fields query takes the resources type as a parameter, not the relation name.
This PR should fix this behaviour.

### CODE REVIEW

**Comment by zacharygolba** [comment]
👍  looks/works great! We'll be able to squash and merge after you rebase. Thank you.

**Comment by zacharygolba** [comment]
It looks like this needs rebased again before we can merge. Ping me when you get this up to date so we can merge this ASAP. 😃

File: package.json [hunk 1]
```diff
@@ -44,7 +44,7 @@
     "inflection": "1.10.0",
     "moment": "2.14.1",
     "ora": "0.2.3",
-    "rollup": "0.34.1",
+    "rollup": "0.34.0",
```
Comment (zacharygolba) [inline package.json]: I think we want to keep rollup at version `0.34.1`.

**Comment by zacharygolba** [comment]
We have to upgrade rollup to `0.34.1` in this PR as well. After that we're good to squash and merge. 😃

**Comment by zacharygolba** [comment]
Looks good just needs rebased again since we merged #217.

**Comment by kureuil** [comment]
Should be good to merge now.

**Comment by zacharygolba** [comment]
👍  Awesome work!

**Pull request merged**